ruby/alias_manager.rb

```diff
@@ -0,0 +1,51 @@
+# for vowels/consonants - compare letter to v/c index

```

Comment:
@Joel-B-Williams This is a solid solution.  I like that you are already starting to encapsulate your work as well.

Lets consider some further design to separate the larger jumble method even more.